### ISSUE TITLE
[Question] should we also do CI builds of Dosbox-SVN? as a test/compare base

### ISSUE THREAD

**Issue by LowLevelMahn** [dosbox-staging/dosbox-staging#1378]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

git

### Whats your question and how can we help?

is there any interest in building Dosbox-SVN (branch) on the CI? so that there is always a vanilla dosbox version for comparison available?

im building Dosbox-SVN locally with VS2019 (and self build dependent libs (SDL,pdcurses,...) and could maybe switch that to use vcpkg

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by arrowgent** [comment]
i use this for frequent svn builds

https://code.launchpad.net/~i30817/+archive/ubuntu/dosbox-patched

though i normally have it disabled because it would update nearly everyday and i dont use it that often...

**Comment by LowLevelMahn** [comment]
i would prefer a CI build of dosbox-staging SVN-Branch - build on the very same CI as the staging releases
just to be safe that everything is as vanilla as possible

im building Dosbox-SVN under Linux/Windows i always by myself - but a CI build could be a better reference for other testers that are not able to build it

**Comment by kcgen** [comment]
I can see the benefits from a testing perspective and to help those authoring patches in general (to SVN and multiple forks). 

I would certainly be open to an `SVN builds` workflow, provided it:
 - Triggers on pushes to the `svn/trunk` branch
 - Informs users that it's intent is for development / testing / QA purposes for the Staging project only
 - Points readers to the DOSBox project website and vogons, for supported DOSBox releases and sources
 - Instructs users to not report issues against against this specific build: instead, they need to use a supported build (from whatever project they're using), downloaded from an official site.
 
I know all of our hearts are in the right place - and we're just trying to further DOSBox in general.  But my biggest fear is is that this comes back to bite us, in the form of users reporting false-positives on vogons, and then core team members have to ask: _"Ok, you say you're running an SVN build, but first we need to ask where you got it."_ 

Perhaps this could be a requirement of this PR workflow (whoever might want to take it up): that safeguards are put in place to limit or restrict visibility of this build - so not only do we have textual warnings, but also technical safeguards to limit exposure of the build.

**Comment by Grounded0** [comment]
As we're getting along well with the DOSBox team we must tread carefully here as stated by @kcgen.

**Comment by LowLevelMahn** [comment]
@kcgen 
fully understandable

sad that even stuff like auto-builts "can" become a problem

> "Ok, you say you're running an SVN build, but first we need to ask where you got it."

and it still a difference if i say: "build by myself" or "from the dosbox staging project" :/

**Comment by arrowgent** [comment]
yes i can see it would be beneficial as some baseline test for staging's changes

ive seen other repos build their plugin library against a main repo and keep a local backup ci
those are more lined up with specific release versions x.x.x-ci, not a constant flood of svn changes or even daily ones.

**Comment by LowLevelMahn** [comment]
> ive seen other repos build their plugin library against a main repo and keep a local backup ci
> those are more lined up with specific release versions x.x.x-ci, not a constant flood of svn changes or even daily ones.

i don't understand this sentence fully: what is a plugin library in context of dosbox? what do you mean with build against a main repo?

currently the offical releases under: https://github.com/dosbox-staging/dosbox-staging/releases
recent development builds can be found here: https://dosbox-staging.github.io/downloads/devel

the same could be done for the dosbox-svn source - ignoring kcgens concernes :)
and it would be the same - official release builds + svn-change builds (the flood)

you talking about archiving some versions or only building selected svn-changes - but who should select them/and why?
